### PR TITLE
Only provision ECR in us-east-1. Other regions will fall back to the …

### PR DESCRIPTION
**Pull request by nzoschke**
In regions where ECR is not provisioned, it will fall back to the registry running in the cluster.

https://github.com/convox/rack/issues/211

### CODE REVIEW

File: api/models/templates/app.tmpl [hunk 1]
```diff
@@ -350,6 +356,7 @@
     "Value": { "Ref": "AWS::AccountId" }
   },
   "RegistryRepository": {
```
Comment (csquared) [inline api/models/templates/app.tmpl]: Does `RegistryId` also need the condition `RegionHasRegsitry`?  

It seems to me if `RegsitryId` is set, and it appears to me to reference something that will always be there, then this branch will still (always) execute, which we do not wish to happen:  https://github.com/convox/rack/blob/master/api/models/build.go#L223 .

**Comment by csquared** [comment]
@nzoschke not sure if this change accomplishes the intended purpose, but think just adding the condition should fix that. lmk if my understanding is off for some reason.

File: api/models/templates/app.tmpl (hunk 1 above)
Comment (ddollar) [inline api/models/templates/app.tmpl]: Yep both of them should have the condition

Comment (nzoschke) [inline api/models/templates/app.tmpl]: Great catch, fixed.

**Comment by ddollar** [comment]
:+1:

**Pull request merged**